lib/trino-parquet/src/test/java/io/trino/parquet/reader/TestInt96Timestamp.java

```diff
@@ -88,6 +98,48 @@ public void testVariousTimestamps()
         testVariousTimestamps(TIMESTAMP_PICOS);
     }
 
+    @Test
+    public void testNanosOutsideDayRange()
+            throws IOException, URISyntaxException
+    {
+        List<String> columnNames = ImmutableList.of("timestamp");
+        List<Type> types = ImmutableList.of(TIMESTAMP_NANOS);
+
+        // int96_timestamps_nanos_outside_day_range.parquet file is prepared with timeOfDayNanos values which are
+        // outside the [0, NANOSECONDS_PER_DAY] range to simulate data generated by AWS wrangler.
+        // https://github.com/aws/aws-sdk-pandas/issues/592#issuecomment-920716270
```

Comment:
that comment suggests it might be a bug in pyarrow.
Will it be fixed there?

Reply:
I believe it's tackled by providing a flag to make awswrangler stop writing int96 timestamps
https://github.com/aws/aws-sdk-pandas/issues/592#issuecomment-922905454
```
wr.s3.to_parquet(..., pyarrow_additional_kwargs={"use_deprecated_int96_timestamps"=False})
```